sampleModel/src/samplemodel/SampleModelNumericPropertyTester.java

```diff
@@ -1,20 +1,20 @@
-package samplemodel;

```

Comment:
Hmm, I can't see what changed in this file.  If it was just whitespace due to reformatting, it may have been better to "discard" the changes.  Happy to talk this through further with you, if you want.